### PR TITLE
Startup: Load open pulls from the DB

### LINKED ISSUE #60: Pulls: load pulls from DB on startup.
We used to load all the open pulls from the DB upon startup and then refresh them in the background. This prevents pulldasher from starting up blank when it restarts.

It's a simple change and probably just got dropped in all the refresh refactoring.

### PR DESCRIPTION
**Pull request by danielbeardsley**
So we don't start out blank.

There's a possibility that we'll load pulls that have been
closed in the meantime, but it's probably better that we
know about them anyway.

Closes #60

### CODE REVIEW

**Comment by copperwall** [comment]
This looks good to me. CR :fire:

**Comment by danielbeardsley** [comment]
Made more commits after actually testing this. Might want to re-CR since I can't be trusted :-)

File: bin/pulldasher [hunk 1]
```diff
@@ -5,7 +5,6 @@ var debug = require('debug')('pulldasher:bin');
 
 // Save the pid to a file if requested (for an init script)
 if (config.pidFile) {
-   fs.writeFileSync(config.pidFile, process.pid);
```
Comment (copperwall) [inline bin/pulldasher]: Why are we getting rid of the pidFile functionality?

Comment (danielbeardsley) [inline bin/pulldasher]: Cause I'm dumb! Fixing.

**Comment by copperwall** [comment]
Cool! CR :fire:

**Pull request merged**